requirements.txt

```diff
@@ -7,4 +7,5 @@ trafilatura
 langchain-huggingface
 qdrant-client
 langchain-qdrant
-numpy==1.26.4
\ No newline at end of file
+numpy==1.26.4
+azure-search-documents==11.5.1
```

Comment:
Revert this. `AzureAISearch` will remind users to install the package if they use it. This could help keep `knowldge-storm` light.

Reply:
have made this optional + added reminder as suggested